src/content/docs/change-tracking/change-tracking-webhooks.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Notify your team about deployments'
+tags:
+  - APM
+metaDescription: "After you set up the changes you want to monitor, you can use a webhook to notify your colleagues about the impacts of those changes."
+---
+
+
+import webHookTest from 'images/tracking_screenshot-crop_webhook_test.webp'
+
+
+
+After you record a deployment for an APM application entity, whether it's with the change tracking feature or the older [REST API](/docs/apm/apm-ui-pages/events/record-deployments/#api-instructions), you can notify your team members using a webhook.
+
+## Get your webhook destination URL [#get-webhook-url]
+
+You can send deployment data to a variety of webhook destinations. Follow the instructions for the tool you're using to get the webhook URL.
+
+If you're using Slack, use the legacy New Relic alerts app:
```

Comment:
Why is Slack the only tool mentioned here? What other tools could be included? Is this pathway listed anywhere else in the docs and can we link to it instead?

Reply:
@homelessbirds Slack seems to be an exception, which might be why it was called out.  I can check with the change tracking team. Alas, the section does tell users to follow the instructions in their tool to get the webhook URL. Slack just happens to be an exception so we give them the details here.  Still, I'll check with the team about this discrepancy.

Reply:
Check out the latest changes to the wording.

Reply:
Much better!